projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -171,6 +155,46 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     @ViewChild(IgxForOfDirective, { read: IgxForOfDirective, static: true })
     protected virtDir: IgxForOfDirective<any>;
 
+    /**
+     * Set the combo's overlaySettings that control how the list of items is displayed.
+     *
+     * ```html
+     * <igx-combo [overlaySettings] = "customOverlaySettings"></igx-combo>
+     * ```
+     *
+     * ```typescript
+     *  const customSettings = { positionStrategy: { settings: { target: myTarget } } };
+     *  combo.overlaySettings = customSettings;
+     * ```
+     */
+    @Input()
+    public set overlaySettings(val: OverlaySettings) {
+        if (val) {
+            const newSettings = val;
+            if (val.positionStrategy) {
+                newSettings.positionStrategy = val.positionStrategy.clone();
+            }
+            Object.assign(this._overlaySettings, newSettings);
+        } else {
+            console.warn('Please provide valid overlay settings');
+        }
+    }
+
+    /**
+     * Get the combo's overlaySettings.
+     *
+     * ```typescript
+     *  const comboOverlaySettings: OverlaySettings = myCombo.overlaySettings;
+     * ```
+     */
+    public get overlaySettings(): OverlaySettings {
+        return this._overlaySettings;
+    }
+
+    public get element() {
```

Comment:
Pretty sure this doesn't need this [to be public], users can ref this as ViewChild